src/exporter.js

```diff
@@ -124,6 +125,10 @@ export default class {
   }
 }
 
+export const SPACE_REPLACER = String.fromCharCode(160);
```

Comment:
> But then anki displays two tags vocab and check_with_tutor. I check it before

Thanks for clarifying.

I think I have a better idea: 

```js
':tags': typeof tags === 'string' ? tags : tagsToStr(tags) // tagsToStr replaces to '_'
```

So the end user of the module will have a choice to use a custom separators